### ISSUE TITLE
Return item kit always adds an whole extra item kit

### ISSUE THREAD

**Issue by AndreasJuulHansen** [opensourcepos/opensourcepos#3957]
### Background information

__IMPORTANT: If you choose to ignore this issue report template, your issue will be closed as we cannot help without the requested information.__

Please make sure you tick (add an x between the square brackets with no spaces) the following check boxes:

- [x] Reporting an issue of an unmodified OSPOS installation
- [x] Checked [open and closed issues](https://github.com/opensourcepos/opensourcepos/issues?utf8=%E2%9C%93&q=is%3Aissue) and no similar issue was already reported (please make sure you searched!)
- [x] Read [README](https://github.com/opensourcepos/opensourcepos/blob/master/README.md), [WHATS_NEW](https://github.com/opensourcepos/opensourcepos/blob/master/WHATS_NEW.txt), [INSTALL.md](https://github.com/opensourcepos/opensourcepos/blob/master/INSTALL.md) and [UPGRADE](https://github.com/opensourcepos/opensourcepos/blob/master/UPGRADE.txt)
- [x] Read the [FAQ](https://github.com/opensourcepos/opensourcepos#faq) for any known install and/or upgrade gotchas (in specific PHP extensions installed)
- [x] Read the [wiki](https://github.com/opensourcepos/opensourcepos/wiki)
- [x] Executed any database upgrade scripts if an upgrade pre 3.0.0 (e.g. database/2.4_to_3.0.sql)
- [x] Aware the installation code that [GitHub master](https://github.com/opensourcepos/opensourcepos/tree/master) is for [developers only](https://github.com/opensourcepos/opensourcepos/wiki/Development-setup) and therefore not complete nor stable.

### Installation information

- OSPOS version is: 3.3.9
- OSPOS git commit hash is: Installed with docker downloaded here from git
- PHP version is: Installed with docker
- MySQL or MariaDB version is: Installed with docker
- OS and version is: Windows 11 23H2 - 22631.3296
- WebServer is: Installed with docker
- Selected language is: English
- (If applicable) Docker installation: v4.28.0
- (If applicable) Installation package for the LAMP/LEMP stack is: 

### Issue / Bug  / Question / New Feature

When I have created an item as an example, a can of Red Bull with all the nessesary information and barcode. I then create an item kit with 24 Red Bull cans in and the other barcode on the package itself. Then when you go into the sales module, and scan the package it puts 24 Red Bull cans in the basket. Then if I go to the return function, and try to return the item kit with the barcode and the 24 cans it it, it just adds 24 extra - giving a total of 48. I can only return a whole package if i scan a Red Bull can 24 times, since the system can't subtract item kits since it always adds item kits in the return menu.

Thanks for a nice product tho!!!!

**Comment by odiea** [comment]
Did you complete the first sale. Then do a Return?

**Comment by AndreasJuulHansen** [comment]
@odiea I have made an order, where I have put in different items. I then suspend the order, and reopen it again some time later. Then I choose the return function, so itwill subtract items. Then I want to make an invoice. I works fine with single items, but not with item kits.

I do it this way, since we buy alcohol in lager quantities at my dorm. Then people say what they want for a party they are helding in our party hall. They get the items, host the party, and return the unopened items afterwards, and then they pay for what they have used. Thats why I have tried to do it with the suspend function. And where it does not work is when I scan a crate of Red Bull and they return a whole crate again, it can't subtract the 24 single Red Bull cans it added when I scanned the crate barcode to begin with.
